content/lessons/table_of_contents.md

```diff
@@ -0,0 +1,86 @@
+# Lessons have the following content & introduced concepts:
```

Comment:
Spodziewałem się, że ta tabelka będzie 1:1 odwzierciedlać listę lekcji. Ale nie zgadza się to z "List of lessons" z https://mimuw-jnp2-rust.github.io/ . To czego konkretnie to jest "table of content"?

Mam też doświadczenie, że utrzymywanie table of content jest trochę trudne. Jak się zmieni jedną lekcję jakoś bardziej drastycznie, to czasem wtedy i table of content przydałoby się zmienić, ale czasem się o tym zapomina... Ale tak bardziej po prostu sobie narzekam, myślę że dobrze mieć jakiś taki overview.

Reply:
To było stworzone przeze mnie luźno rok temu. Wrzuciłem, byś miał co najmniej tyle materiałów, ile ja miałem ostatnio,.